FWCore/Framework/test/dependentrecord_t.cppunit.cc

```diff
@@ -38,6 +38,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <cstring>
```

Comment:
While you're doing changes, could you remove this include as it should not be needed now?